test/e2e/e2e_test.go

```diff
@@ -11,15 +11,23 @@ import (
 	"testing"
 	"time"
 
-	"github.com/openshift/service-ca-operator/pkg/controller/api"
-	"github.com/openshift/service-ca-operator/pkg/operator/operatorclient"
-
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	pkgruntime "k8s.io/apimachinery/pkg/runtime"
```

Comment:
nit: `kruntime`?

Reply:
Done

`pkgruntime` is the alias I've commonly seen in k/k and related upstream projects, I guess `kruntime` is the openshift version?